scss-partials/_general.scss

```diff
@@ -0,0 +1,265 @@
+    }
+
+    &__column {
+        width: 33.33%;
+    }    
+}
+
+.logo {
+    text-align: center;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    &__bow {
+        height: 10px;
+        width: 19px;
+        margin: 12px 0px 0px;
+    
+        @include media ( ">=lg" ) {
+            height: 19px;
+            width: 37px;
+            margin: 16px 0px 0px;
+        }
+    }
+    
+    &__name {
+        width: 117px;
+        height: 6.5px;
+        margin: 7px 0px 0px;
+    
+        @include media ( ">=lg" ) {
+            width: 220px;
+            height: 12px;
+            margin: 13px 0px 0px;
+        }
+    }
+}
+
+.control {
+    text-align-last: right;
+
+    &__option {
+        text-align: center;
+        position: relative;
+        display: inline-block;
+        cursor: pointer;
+        height: 20px;
+        text-align: center;
+        margin: 14px 56px 0px 0px;
+    
+        @include media ( ">=md" ) {
+            margin: 16px 56px 0px 0px;
+        }
+    
+        @include media ( ">=lg" ) {
+            margin: 24px 16px 0px 0px;
+            height: 24px;
+        }
+    }
+}
+
+.link {
+    cursor: pointer;
+
+    &:hover { text-decoration: underline; }
+}
+
+.create {
+    background: $tws_blue;
+    color: $tws_white;
+    text-transform: uppercase;
+    width: 94px;
+    padding: 4px 16px;
+    margin-right: 32px;
+
+    &:hover { background: $tws_coral; }
+
+    &__list {
+        text-align: center;
+        font-weight: bold;
+        letter-spacing: 0.4px;
+    }
+}
+
+.create, .findCouple {
+    display: none;
+
+    @include media ( ">=lg" ) {
+        display: inline-block;
+    }
+}
+
+.menu {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0;
+    font-family: $font-regular;
+
+    @include media ( "<lg" ) {
+        display: none;
+    }
+
+    &__list {
+        display: flex;
+        justify-content: space-evenly;
+        width: 100%;
+        padding: 0px;
+        margin: 0px;
+    
+        &--item {
+            margin: 32px 8px 0px;
+            padding: 0px 0px 8px;
+            color: $tws_black;
+            border-bottom: 2px solid $tws_white;
+            letter-spacing: 0.8px;
+            white-space: nowrap;
+            list-style: none;
+            line-height: 20px;
+            cursor: pointer;
+    
+            &:hover { border-bottom: 2px solid $tws_coral; }
+        }
+    }
+}
+
+.search__img {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
+    margin: 17px 0px 7px 17px;
+
+    @include media ( ">=md" ) {
+        margin: 21px 0px 7px 17px;
+    }
+    
+    @include media ( ">=lg" ) {
+        margin: 33px 0px 16px 33px;       
+    }
+}
+
+.cta {
+    align-items: center;
+    text-align: center;
+    margin: 100px 0px 48px;
+
+    @include media ( "<=sm" ) {
+        margin: 64px 0px 0px;
+    }
+    
+    &__header {
+        color: $tws_black;
+        font-family: $font-heading;
```

Comment:
I've noticed the fonts look slightly different from the ones on production even though styles seem correct. You might want to add the below to body styles, perhaps it will solve it
```
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
```

Reply:
I'm not sure I can see any difference, shall I check the font etc on live instead of the figma design?